### PR TITLE
Grant #006 - Swap & Liquidity iFrame-FrontEnd

### PR DESCRIPTION
**Pull request by TF-Gordon**
@obiedobo 
[Grant #006](https://github.com/ApeSwapFinance/apeswap-frontend/issues/813)

[Gordon](https://discord.com/users/842553249461698572)
Wallet Address: 0x994C5FA754E4370a7EAc140332585FF8009b9ddf

Updated the ApeSwap FrontEnd to check if a window is an iframe and to hide header elements and footer. Kept the existing styling and just hid the header buttons to have a consistent look and feel on the site. Also had to make modifications to the UIKit to pass the iframe check as a parameter in order to hide the footer. Created a pull request for the UIKit as well.

Copy and paste code below into an html file then open it while front end is running. It will display site in an iframe with the header elements and footer hidden.
```
<html>
<body>
TEST
<br/>
<iframe src="http://localhost:3000/orders" width="1500px" height="600px" />
</body>
</html>
```

### CODE REVIEW

**Comment by TF-Gordon** [comment]
Also see UIKit pull request [https://github.com/ApeSwapFinance/apeswap-uikit/pull/101]()

**Comment by obiedobo** [comment]
I've updated the package.json with a deployed version of your UI kit! Testing pretty well 😋

Would you be able to merge the latest from staging into your branch & then I'll get a devs eyes on it?

**Comment by TF-Gordon** [comment]
@obiedobo Sure just completed the merge from staging into my branch.

**Comment by obiedobo** [comment]
Apologies for the delay & thanks for keeping this fresh. We had some folks out last week!

My senior dev is working on delivering a major push on Tuesday, which is taking the majority of his bandwidth. But this is on his radar!

**Comment by TF-Gordon** [comment]
@obiedobo Got it. I saw it had an error again in the workflow. I ran the yarn install command and committed the updated code hopefully that should have it up to date now.

**Comment by TF-Gordon** [comment]
@obiedobo Since pulling the latest code Vercel is giving me a 401 for the SDK that's not something i've touched though. Is this a problem or will it fix itself?

**Comment by Harambe-Nakamoto** [comment]
I dont think it should be giving an error for the SDK. Is it still giving you that error? Since your not authorized for automatic deploys Vercel should be giving you an unauthorized error

**Comment by Harambe-Nakamoto** [comment]
Looks good so far! How hard would it be for you to remove the navbar from the iframe as well? Don't worry about it if you think it would be a large lift

**Comment by TF-Gordon** [comment]
@Harambe-Nakamoto I can remove the nav bar but since it contains the connect wallet button then that would need to be implemented again. I was trying to implement the iFrame in a way that doesn't require up keep when there are changes to the site. I think removing the nav bar would require that button to be implemented on every page when using iFrame which might add a bit of overhead in checking the load each time.

**Comment by Harambe-Nakamoto** [comment]
Since the swap prompts the user to connect when they are not connecting I think we could get away without having the navbar unless there are side effects that I am not thinking of.

**Comment by TF-Gordon** [comment]
@Harambe-Nakamoto Yea that make sense. I'll start working on that then and will let you know once I have it ready for testing.

**Comment by Harambe-Nakamoto** [comment]
@TF-Gordon Thank you! I will get that reviewed and published

File: src/components/Menu/index.tsx [hunk 1]
```diff
@@ -15,9 +15,11 @@ import bscConfig from './chains/bscConfig'
 import maticConfig from './chains/maticConfig'
 import { languageList } from '../../config/localization/languages'
 import ethConfig from './chains/ethConfig'
+import iframeConfig from './chains/iframeConfig'
 import MoonPayModal from '../../views/Topup/MoonpayModal'
 
 const Menu = (props) => {
+  const isIframe = window.self !== window.top
```
Comment (Harambe-Nakamoto) [inline src/components/Menu/index.tsx:22]: Could we put this in a try, catch since some browsers block window.top? After that this is good!

Comment (TF-Gordon) [reply src/components/Menu/index.tsx]: @Harambe-Nakamoto Just pushed the change. Let me know if everything looks ok.

**Review by Harambe-Nakamoto — APPROVED** [review]
Nice work @TF-Gordon!
@obiedobo This is approved and ready for a quick final test

**Comment by TF-Gordon** [comment]
@Harambe-Nakamoto looks like its still erroring 401 unauthorized when lint is installing the dependencies. I haven’t made any changes related to that.

**Pull request merged**